lib/react.dart

```diff
@@ -44,11 +61,15 @@ abstract class Component {
   /// TODO: Switch back to a plain field once this issue is fixed.
   Ref _ref;
 
-  /// The React context map of this component, passed down from its ancestors' [getChildContext] value.
-  ///
-  /// Only keys declared in this component's [contextKeys] will be present.
-  Map get context => _context;
-  set context(Map value) => _context = value;
+  /// The React context value of this component, passed down from its Provider.
+  dynamic get context => _context;
+  set context(dynamic value) => _context = value;
+
+  /// The contextType property on a class can be assigned a Context object created by createContext().
+  /// This lets you consume the nearest current value of that Context type using this.context.
+  /// You can reference this in any of the lifecycle methods including the render function.
+  ReactDartContext get contextType => _contextType;
```

Comment:
Since this is only accessed once per component class, there shouldn't be a need for a setter or this local variable.